extension/js/agent/patches/patchBackboneComponent.js

```diff
@@ -38,7 +38,7 @@ var patchBackboneComponent = _.bind(function(BackboneComponent, instancePatcher)
         // la proprietà sarà ereditata anche dai sottotipi e finirà nelle varie istanze,
         // contiene la versione patchata della initialize
 
-        var initializeFncName = _.has(BackboneComponent.prototype, 'initialize') ? 'initialize' : 'start';
```

Comment:
in 2.3 Application always has `initialize`defined but  `_.has` doesn't find it cause it's shallow